### ISSUE TITLE
Restore Migration Analytics features when new backend implementation is settled

### ISSUE THREAD

**Issue by mturley** [kubev2v/forklift-ui#281]
This issue is for the eventual reversion of PR https://github.com/konveyor/virt-ui/pull/282 (issue #274).

**Comment by mturley** [comment]
One thing to note when we reimplement this: Polling was disabled in the plan wizard due to issues it may cause in the mapping steps. I think that may have been resolved now by our changes for editing plans, which made those steps more resilient. We will probably want to turn polling back on in the wizard, at least on the Select VMs step, in order to make sure the analytics data is always up to date (especially since we added a spinner when the concerns are out of date in https://github.com/konveyor/forklift-ui/pull/242).
